src/txn/txn_rollback_to_stable.c

```diff
@@ -867,7 +863,11 @@ __wt_rollback_to_stable(WT_SESSION_IMPL *session, const char *cfg[])
      * concurrently.
      */
     WT_RET(__wt_open_internal_session(S2C(session), "txn rollback_to_stable", true, 0, &session));
+
+    F_SET(session, WT_SESSION_RTS);
```

Comment:
roll back to stable is single threaded? It is out of scope, can we parallelise that for different tables?

Reply:
Yes, rollback to stable is currently single-threaded. Maybe in future, we can parallelize it if we are finding the performance problems with a single thread.